### ISSUE TITLE
where do I learn about DIC

### ISSUE THREAD

**Issue by beneisdorfer** [iDICs/forum#1]

**Comment by dicengine** [comment]
Here's a great resource for you: https://idics.org/guide/

**Comment by iadicola** [comment]
Don't forget the DIC 101 class [https://youtu.be/OBveur2gNdI] now offered on line.  Or take it in-person at the 2022 meeting.

**Comment by PhillipReu** [comment]
Yes. The in person conference will be in Nantes. https://pm-idics-2021.org/
